kuma/javascript/src/active-banner.jsx

```diff
@@ -232,23 +232,11 @@ export default function ActiveBanner() {
                         id={id}
                         classname="mdn-subscriptions"
                         title={gettext('Become a monthly supporter')}
-                        copy={
-                            <Interpolated
-                                id={gettext(
-                                    'Support MDN with a $5 monthly subscription <learnMore/>.'
-                                )}
-                                learnMore={
-                                    <a href={`/${locale}/payments/`}>
-                                        {gettext('Learn more')}
-                                    </a>
-                                }
-                            />
-                        }
-                        cta={gettext('Subscribe')}
-                        url={`${
-                            window.mdn ? window.mdn.wikiSiteUrl : ''
-                        }/${locale}/profiles/${userData.username ||
-                            ''}/edit#subscription`}
+                        copy={gettext(
+                            'Support MDN with a $5 monthly subscription'
```

Comment:
@schalkneethling Since you're going to have to merge conflict update this anyway, do you think you can squeeze in a jsx code comment that mentions `https://github.com/mdn/kuma/issues/6654` here?